### ISSUE TITLE
Create some tests that can write transactions to Testnet and/or MainNet

### ISSUE THREAD

**Issue by msgilligan** [msgilligan/bitcoin-spock#23]
Specifically, with a hot wallet of (Test)BTC and MSC and/or TMSC, it would be nice to be able to take some/all of the transactions in the STO TSV file and put them out on TestNet and/or MainNet.

**Comment by dexX7** [comment]
It's an interesting idea, especially because those "test transactions" could be used as "checkpoints" within Master Core.

The block generation would have to be replaced with some "blockchain watcher" and a callback that is triggered, once a new block is mined and the previous steps confirmed.

A very important point in my opinion would be to prevent accidental execution of such a "test" ... which once happend to me with some older Master Core regtests that started a bunch of mastercored instances, but used the default bitcoin.conf. Luckily this was on testnet and the only notable effect was to end up with a lot of junk addresses in the wallet.. :)

**Comment by msgilligan** [comment]
Yeah, I'm not sure how we could make a test fixture (Spock extension?) that could match "when" clauses with "then" clauses after the appropriate blocks have been generated. I should pose that on the Spock list.

Anyway, this is lower priority for me than finishing the RegTest stuff, but I wanted to create the issue so we can get back to it later.

**Comment by dexX7** [comment]
I was actually thinking about tweaking `generateBlock()` somehow, such that it doesn't generate a block, but queries the blockchain and returns, once a new block is mined.
